.github/workflows/tests.yml

```diff
@@ -172,14 +172,6 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Set temp dirs correctly
-        shell: cmd
-        # https://github.com/actions/virtual-environments/issues/712
-        run: |
-          echo "TMPDIR=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
```

Comment:
we don't need to add these in env variables for windows anymore???

Reply:
`%TEMP%` and `%TMP%` are default variables on windows, this was added to address the issue here https://github.com/actions/virtual-environments/issues/712 which is hopefully a non-issue today (that's a 2 year old issue)

Reply:
ok.. then removing that should sound good